Shopipy.ProductManagement/Controllers/ProductVariationController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Shopipy.ProductManagement.DTOs;
+using Shopipy.ProductManagement.Services;
+using Shopipy.Persistence.Models;
+using Shopipy.Shared;
+using Shopipy.Shared.Services;
+
+namespace Shopipy.ProductManagement.Controllers
+{
+    [Route("{businessId}/products/{productId}/variations")]
```

Comment:
https://github.com/imbieras/shopipy/blob/0523318744943e2964beba1a74ff2fd0e963b0b3/Shopipy.CategoryManagement/Controllers/CategoryController.cs#L11 or any other Management. Basically this doesn't follow the convention.